NOTICE

```diff
@@ -34,15 +34,15 @@ Copyright The Apache Software Foundation
 Netty
 Copyright The Netty Project
 
-This product includes additional software licensed under the terms
-of the following licenses:
+This product may include or have transitive dependencies to additional software
```

Comment:
IMHO `may include` is too weak. Downstream projects are required to copy this NOTICE. I think it has to be exact.

Reply:
It's a warning to look at the transitive dependencies as well.

Reply:
As this `NOTICE` is about source distribution (not binary distribution), we should only consider code copied from other Apache licensed projects.
So, it means the `NOTICE` should contains `NOTICE` from:
* Netty
* Apache Polaris
* Apache Iceberg
NB: Gradle doesn't provide any `NOTICE`.

It's not required to mention something.